lib/game.rb

```diff
@@ -0,0 +1,84 @@
+#no structure yet, just drafting some logic
+#will split up into dif methods in the future
+
+class Game
+  attr_reader :guess
```

Comment:
You can put all the attr_reader values on the same like eg: `attr_reader :code, :guess`